components/Button/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "main": "Button.js"
```

Comment:
I took this component's (Button and ExitPreviewButton) from docsV3 and that's how they're built.This helps us to avoid having the component filename be index.js